backend/app/fields.py

```diff
@@ -0,0 +1,27 @@
+"""
+Custom fields for the gender analysis web app.
+"""
+from django.db import models
+from django.utils.translation import gettext_lazy
+
+
+class LowercaseCharField(models.CharField):
+    """
+    A subclass of models.CharField that converts the string to lowercase
+    when saving the model instance to the database.
+    """
+
+    # WORK IN PROGRESS!
+    description = gettext_lazy(f"String (up to {super().max_length}) \
```

Comment:
I think "String (up to length 5) ..." makes more sense.

Reply:
I've refactored this so that it hopefully reads easier, but why 5 specifically? I decided to use `self.max_length` since the maximum length of the string the `LowercaseCharField` instance represents is determined by the value the user passes in when creating a Model class that uses this field (in my case, the `max_value` is 40, but it may be different if others also decide to use the `LowercaseCharField`.

Reply:
yes I agree with Joshua! It's more modular in my opinion!